### PR TITLE
Update to Render 8.x

### PR DESCRIPTION
**Pull request by arturpimentel**
#### What is the purpose of this pull request?
Update App Store to the latest Render framework

#### What problem is this solving?
This updates very old/deprecated/unmaintained dependencies and also is the first step into making the App Store compliant to with the newest VTEX IO development standards

#### How should this be manually tested?
1. Make a new dev workspace in account `extensions`
2. Link the app and navigate the store (`https://{{your-workspace}}--extensions.myvtexdev.com`)

#### Screenshots or example usage
None

#### Types of changes

* [ ] Bug fix (non-breaking change which fixes an issue)
* [x] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 node_modules/
 .DS_Store
+*/**/yarn.lock
```
Comment (klzns) [inline .gitignore]: Why is that necessary?

Comment (arturpimentel) [reply .gitignore]: I just thought that `yarn.lock` often pollutes git diffs for no real benefit -- we aren't versioning `node_modules`, therefore we have to run yarn anyway, often generating/overwriting the local version.

Comment (arturpimentel) [reply .gitignore]: But on second thought... https://yarnpkg.com/blog/2016/11/24/lockfiles-for-all/ 🤔

File: manifest.json [hunk 2]
```diff
@@ -4,22 +4,20 @@
...
+    "vtex.styleguide": "9.x",
+    "vtex.store-header": "2.x",
+    "vtex.store-footer": "2.x",
+    "vtex.product-details": "1.x"
   },
   "builders": {
-    "pages": "0.x",
-    "react": "2.x"
+    "messages": "1.x",
+    "pages": "1.x",
```
Comment (brunojdo) [inline manifest.json]: store: 1.x

Comment (arturpimentel) [reply manifest.json]: Done
